src/aae_exchange.erl

```diff
@@ -237,8 +245,8 @@ start(BlueList, PinkList, RepairFun, ReplyFun) ->
 
 -spec start(exchange_type(),
             input_list(), input_list(),
-            fun((list({any(), any(), any()})) -> ok),
-            fun(({closing_state(), non_neg_integer()}) -> ok),
+            repair_fun(),
```

Comment:
This type is different ?? from a single 3-tuple to a 2-tuple of 2-tuples. Is that on purpose?

Reply:
sorry, I got the original type wrong (although dialyzer didn't seem to notice).  The new one is correct - it is a 2-tuple of 2-tuples